fdbserver/workloads/ApiCorrectness.actor.cpp

```diff
@@ -114,6 +132,11 @@ struct ApiCorrectnessWorkload : ApiWorkload {
 		int maxTransactionBytes = getOption(options, "maxTransactionBytes"_sr, 500000);
 		maxKeysPerTransaction = std::max(1, maxTransactionBytes / (maxValueLength + maxLongKeyLength));
 
+		validateEncryptionAtRest =
+		    g_simulator->isSimulated()
+		        ? getOption(options, "validateEncryptionAtRest"_sr, deterministicRandom()->coinflip() ? 1 : 0)
```

Comment:
Unless this is causing a heavy increase in test durations is there any reason we can't always make it true?

Reply:
Idea is to add some `buggify` like semantics unless clients explicitly want it to be enabled all the time.